PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-facet/src/main/java/com/microsoft/azure/toolkit/intellij/facet/projectexplorer/ConnectionNode.java

```diff
@@ -34,13 +42,23 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Objects;
+import java.util.*;
+
+import static com.microsoft.azure.toolkit.intellij.connector.ResourceConnectionActionsContributor.EDIT_CONNECTION;
+import static com.microsoft.azure.toolkit.intellij.connector.ResourceConnectionActionsContributor.REMOVE_CONNECTION;
 
 public class ConnectionNode extends AbstractTreeNode<Connection<?, ?>> implements IAzureFacetNode {
     private final AzureModule module;
+    private final Action<?> editAction;
 
     public ConnectionNode(@Nonnull Project project, @Nonnull AzureModule module, @Nonnull final Connection<?, ?> connection) {
         super(project, connection);
         this.module = module;
+        this.editAction = new Action<>(Action.Id.of("user/connector.edit_connection_in_editor"))
+                .withLabel("Edit Azure Connection in Editor")
```

Comment:
should be `Azure Resource Connection` or `Resource Connection` or `Connection`

Reply:
-> open in editor